### PR TITLE
tests: sign new nested-18|20* models to allow for generic serials

### PR DESCRIPTION
**Pull request by pedronis**

### CODE REVIEW

**Review by mvo5 — APPROVED** [review]
Looks fine

File: tests/lib/assertions/CA5GLZgNQWPhspDQK63Er46Uxz2SO7ez.auto-import.assert.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "type": "system-user",
+    "authority-id": "CA5GLZgNQWPhspDQK63Er46Uxz2SO7ez",
+    "series": ["16"],
+    "brand-id": "CA5GLZgNQWPhspDQK63Er46Uxz2SO7ez",
+    "email": "snappy-dev@lists.launchpad.net",
+    "models": ["pc"],
```
Comment (anonymouse64) [inline tests/lib/assertions/CA5GLZgNQWPhspDQK63Er46Uxz2SO7ez.auto-import.assert.json:7]: the existing auto-import.json has `pc`,  `pc-i386` and `pi2` models in it too, perhaps @sergiocazzolato could comment if this model you are adding should also have those models listed here too?

**Review by anonymouse64 — APPROVED** [review]
LGTM, but perhaps @sergiocazzolato could comment as well?

File: tests/lib/assertions/CA5GLZgNQWPhspDQK63Er46Uxz2SO7ez.auto-import.assert.json (hunk 1 above)
Comment (pedronis) [reply tests/lib/assertions/CA5GLZgNQWPhspDQK63Er46Uxz2SO7ez.auto-import.assert.json:7]: I have signed with my key so far only "pc"-named models,  it needs updating if I do sign some of those other models, but I prefer to extend the list only when needed

**Pull request merged**